.github/workflows/release.yml

```diff
@@ -11,6 +11,10 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
     environment: release-recover-id-object
```

Comment:
The environment should be renamed if we intend to use this release with a matrix and several services.

Reply:
Will do that in a new PR when I actually am ready to release. :)

Reply:
I will remove matrix changes until it may be an issue.